backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/theme/BpkTheme.kt

```diff
@@ -37,34 +48,76 @@ fun BpkTheme(
   content: @Composable () -> Unit,
 ) {
   val typography = BpkTypography(defaultFontFamily = fontFamily)
+  val colors = if (isSystemInDarkTheme()) BpkColors.dark() else BpkColors.light()
+
   CompositionLocalProvider(
     LocalBpkTypography provides typography,
+    LocalBpkColors provides colors,
+    LocalElevationOverlay provides null,
```

Comment:
Is this intentional or missing?

Reply:
Its intentional behaviour of Material design system. But we don't rely on it really :)